src/java.xml.crypto/share/classes/javax/xml/crypto/dsig/SignatureMethod.java

```diff
@@ -235,6 +235,28 @@
      */
     String HMAC_SHA512 = "http://www.w3.org/2001/04/xmldsig-more#hmac-sha512";
 
+
+    /**
+     * The <a href="http://www.w3.org/2007/05/xmldsig-more#rsa-pss">
+     * RSASSA-PSS</a> signature method algorithm URI.
+     * <p>
+     * If the {@code params} parameter is {@code null} when calling
+     * {@link XMLSignatureFactory#newSignatureMethod} with {@code RSA_PSS}
+     * as the signature algorithm, the default parameter as defined in
+     * <a href="https://tools.ietf.org/html/rfc6931#section-2.3.9">RFC 6931 Section 2.3.9</a>
+     * is used and this default parameter will also be returned by the
```

Comment:
WE should mention/link to the type returned. Suggest breaking this into two sentences: 

`If the {@code params} parameter is {@code null} when calling {@link XMLSignatureFactory#newSignatureMethod} with {@code RSA_PSS} as the signature algorithm, the default parameter as defined in <a href="https://tools.ietf.org/html/rfc6931#section-2.3.9">RFC 6931 Section 2.3.9</a> is used. This default parameter is represented as an {@link RSAPSSParameterSpec} type and returned by the {@link SignatureMethod#getParameterSpec()} method.`

Reply:
OK, and some other tweaks. New commit pushed.